### PR TITLE
chunk network resources

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

Updates the session dev tools `network` page to load data from `Traces`.
Updates the Resource context to use traces to load session network data across the product.
Doesn't yet switch to the new traces table to render network resources, but should make it easy to do so in the future.

## How did you test this change?

* tested with otel frontend traces
* tested with frontend network requests recorded as traces

https://www.loom.com/share/3a252f8576e84405aa5ac6ded29868dd

## Are there any deployment considerations?

monitoring in prod, will follow up to clean up network request files

## Does this work require review from our design team?

no

### CODE REVIEW

File: __generated/rr/rrweb/rr.js
Comment (SpennyNDaJets) [inline __generated/rr/rrweb/rr.js:1]: Are these changes anticipated?

**Review by SpennyNDaJets — APPROVED** [review]
Click test looks good to me!

**Review by SpennyNDaJets — CHANGES_REQUESTED** [review]
One thing I noticed is that we are still only returning 50 traces? Looks like the pagination is based on `1_000` but the cursor is not respecting that limit

File: frontend/src/pages/Player/ResourcesContext/ResourcesContext.tsx [hunk 1]
```diff
@@ -98,112 +166,48 @@ const buildResources = (resources: NetworkResource[]) => {
...
+		query: `secure_session_id=${session?.secure_id} AND highlight.type=http.request`,
+		projectId,
+		startDate: moment(session?.created_at).toDate(),
+		endDate: moment(session?.created_at).add(4, 'hours').toDate(),
+		traceCursor: undefined,
+		sortDirection: SortDirection.Asc,
+		sortColumn: 'timestamp',
+		skip: !session?.secure_id,
+		skipPolling: true,
+		limit: 1_000,
```
[the remark this answers is not in the document]
Comment (Vadman97) [reply frontend/src/pages/Player/ResourcesContext/ResourcesContext.tsx:194]: yup meant to say 1000

File: __generated/rr/rrweb/rr.js
Comment (Vadman97) [reply __generated/rr/rrweb/rr.js:1]: unrelated, reverting

**Comment by Vadman97** [comment]
> One thing I noticed is that we are still only returning 50 traces? Looks like the pagination is based on `1_000` but the cursor is not respecting that limit -> think we just need to pass it into `getConnection`

thanks for the callout @SpennyNDaJets and for the help with the fix. just updated this!

**Comment by Vadman97** [comment]
new testing https://www.loom.com/share/8016f403dd6a4ac08a1a61886c8e187d

**Review by SpennyNDaJets — APPROVED** [review]
LGTM

**Pull request merged**